sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/catalog/SessionCatalogSuite.scala

```diff
@@ -465,6 +466,48 @@ class SessionCatalogSuite extends SparkFunSuite {
     assert(plan == SubqueryAlias("range", tmpView, Option(TableIdentifier("vw1"))))
   }
 
+  test("look up view relation") {
+    val externalCatalog = newBasicCatalog()
+    val sessionCatalog = new SessionCatalog(externalCatalog)
+    val metadata = externalCatalog.getTable("db3", "view1")
+    sessionCatalog.setCurrentDatabase("default")
+    // Look up a view.
+    assert(metadata.viewText.isDefined)
+    val view = normalizeExprIds(View(desc = metadata, output = metadata.schema.toAttributes,
+      child = CatalystSqlParser.parsePlan(metadata.viewText.get)))
+    assert(
+      normalizeExprIds(sessionCatalog.lookupRelation(TableIdentifier("view1", Some("db3"))))
+        == SubqueryAlias("view1", view, Some(TableIdentifier("view1", Some("db3")))))
+    // Look up a view using current database of the session catalog.
+    sessionCatalog.setCurrentDatabase("db3")
+    assert(
+      normalizeExprIds(sessionCatalog.lookupRelation(TableIdentifier("view1")))
+        == SubqueryAlias("view1", view, Some(TableIdentifier("view1"))))
+  }
+
+  /**
```

Comment:
The problem is both the `PlanTest` and `SparkFunSuite` are abstract classes, should we make `PlanTest` a trait?